proloauth_client/utils.py

```diff
@@ -25,7 +31,8 @@ def refresh_token(request, user, data):
 
 def update_user(user, data):
     for key in USER_SYNC_KEYS:
-        setattr(user, key, data['user'][key])
+        if not getattr(user, key):
```

Comment:
```suggestion
        if getattr(user, key) is None:
```

Otherwise we will update `False`, empty strings, ...

Reply:
I did not think about `False` ! However, getattr() can return `''` when the field is empty, but not `None`, right ?

Reply:
I think Django (and postgres as a backend) will do a fine job differentiating `None` when a field as no value and `""` for an empty string, you probably don't have to bother with this

Reply:
It seems you're right: https://docs.djangoproject.com/fr/3.0/ref/models/fields/#django.db.models.Field.null

As these values usually come from html forms I guess it makes less sense to type this anyway ....